.github/workflows/sbt-dependency-graph.yaml

```diff
@@ -0,0 +1,31 @@
+name: Update Dependency Graph for sbt
+on:
+  push:
+    branches:
+      - main
+      - sbt-dependency-graph-228eca55167613ca
```

Comment:
I think this line was supposed to be deleted
![image](https://github.com/user-attachments/assets/2ba54b56-d0ae-4c43-9dae-889c6a1aa435)

Reply:
Good catch! Although it's probably not doing any harm?

Reply:
no I think no issue there specifically, I guess the concern I had was whether the second bullet point is important for a multi project build.  I'm not sure if devx expect a line by line comparison (assume not) or just check the main modules are covered.

Reply:
I ran this PR by @rupertbates but not sure he took a deep look into it